static/js/popovers.js

```diff
@@ -1130,8 +1130,10 @@ export function register_click_handlers() {
     });
 
     $("body").on("click", ".flatpickr-confirm", (e) => {
-        const datestr = $(".remind.custom")[0].value;
-        reminder_click_handler(datestr, e);
+        if ($(".remind.custom")[0]) {
+            const datestr = $(".remind.custom")[0].value;
+            reminder_click_handler(datestr, e);
+        }
```

Comment:
I guess ideally we'd change the selector rather than adding this condition; is the problem that the `flatpickr-confirm` element isn't inside any CSS class marking why we have that flatpickr?

Reply:
Yeah, a global event handler on `.flatpickr-confirm` shouldn't exist ideally. Since, reminder popover is currently not active, we can either delete (or comment it out ) or put a condition like this to stop it from running when user clicks on `.flatpickr-confirm`.